Walkthroughs implementation (#1698)
Microsoft/vscode-react-native: media/features-tutor.md

# Learn main features

> ## Commands of the Command Palette

Reviewer: ```suggestion
## React Native commands in the Command Palette
```

> This extension supports many `commands of the Command Palette` for running and debugging React Native applications, and other supporting commands for convenient use of the extension. The complete list of the command you can see in the corresponding [paragraph](https://github.com/microsoft/vscode-react-native#react-native-commands-in-the-command-palette) of the documentation.

Reviewer: ```suggestion
The extension provides special `Command Palette commands` for running React Native applications on different platforms, operating with the Packager and automating some dev processes. You can find more info about supported commands [here](https://github.com/microsoft/vscode-react-native#react-native-commands-in-the-command-palette).
```

## Network inspector

> The extension provides `Network inspector` feature to inspect outgoing network traffic in your apps. You can browse all requests being made and their responses in VS Code DevTools console. Environment requirements and other details for using the Network inspector can be found in [this](https://github.com/microsoft/vscode-react-native#network-inspector) paragraph of the documentation.

Reviewer: ```suggestion
The extension provides `Network inspector` feature to inspect outgoing network traffic in your apps. You can browse all requests being made and their responses in VS Code DevTools console. More details about Network inspector usage and its requirements are available [here] (https://github.com/microsoft/vscode-react-native#network-inspector) in the documentation.
```

## Element inspector

> With the command `reactNative.command.runInspector` you can launch React DevTools to conveniently work with the DOM-tree of your application. Details on how you can install and use React DevTools in this extension can be found [here](https://github.com/microsoft/vscode-react-native#element-inspector).

Reviewer: ```suggestion
In the extension you can run React DevTools Element inspector to inspect the DOM-tree of your application by starting `Run Element Inspector` command in Command Palette. You can learn more about React DevTools installation and usage [here](https://github.com/microsoft/vscode-react-native#element-inspector).
```
